WordPress/wordpress-playground
# Derive MIME types for PHP served files from shared JSON

## What is this PR doing?

This PR eliminates duplication in the code base where we maintained separate MIME type mappings for PHPRequestHandler and for files served via PHP on WP Cloud.

Related to #1197

## How is the problem addressed?

This PR generates `mime-types.php` from shared JSON when deploying the website to WP Cloud.

## Testing Instructions

- Temporarily adjust WP Cloud deploy workflow to be tested under this PR
- Confirm the workflow completes without error and that playground-dot-wordpress-dot-net.atomicsites.blog loads properly
